Dangerfile.swift

```diff
@@ -26,11 +26,15 @@ let prTitle = danger.github.pullRequest.title
 if prTitle.contains("WIP") {
     warn("PR is classed as _Work in Progress_.")
 }
+
 if prTitle.count < 5 {
-    warn("PR title is too short. 🙏 Please use this format `[SDK-000] Your feature title` and replace `000` with Jira task number.")
+    fail("PR title is too short. 🙏 Please use this format `feature/NAME_000_TASK Your feature title` and replace `000` with Jira task number.")
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/dbgarcia/danger-bot/blob/4ad3f65193926f1211a9cfb888b46cbbba684b48/Dangerfile.swift#L31"></span>Line should be 130 characters or less: currently 143 characters<br />
<code>line_length</code> <code>Dangerfile.swift:31</code></del></td>
    </tr>
  </tbody>
</table>